### ISSUE TITLE
Inconsistency between function signatures in a tutorial?

### ISSUE THREAD

**Issue by tatsuyafujisaki** [arrow-kt/arrow#2041]
In this document (https://arrow-kt.io/docs/patterns/monads/#deferred-future), `runSync` is defined as follows.
```kotlin
fun runSync(action: (Try<T>) -> Unit) =
    runBlocking { action(Try { instance.await() }) }
```

Meanwhile, `runSync`, in action, takes a lambda that **takes a map**(=(`city`, `speaker`)) as follows. Doesn't it contradict the function signature above?
```kotlin
repository
    .loadSpeaker()
    .flatMap { speaker -> speaker.nextTalk()
        .flatMap { talk -> talk.getConference() }
        .flatMap { conference -> conference.getCity().map { it to speaker } }
    }
    .runSync { (city, speaker) -> city.fold(
        { ... },
        { ... })
    }
```

**Comment by aballano** [comment]
Hi @tatsuyafujisaki, thanks for opening an issue with your concern! 

Before I tried the code I've seen this docs are about to change as we're removing Try for good, [you can see how's gonna be fixed in this PR](https://github.com/arrow-kt/arrow/pull/1979/files#diff-bf404797c7a09848cf96debfb59d4bd3R480), so we can close this issue afterwards.

**Comment by rachelcarmena** [comment]
Solved via #1979
